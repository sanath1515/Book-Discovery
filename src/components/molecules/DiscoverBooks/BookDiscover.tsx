import { Grid, makeStyles,Button } from "@material-ui/core"
import theme from "../../../theme/theme"
import React from "react";

const useStyles = makeStyles({

  container: {
    position: 'relative',
    width: '100%',
    maxWidth: '400px',
    zIndex: -2
  },

  btn: {
    position: 'absolute',
    top: '80%',
    left: '25%',
    transform: 'translate(-50%, -50%)',
    backgroundColor: ' blue',
    color: 'white',
    fontSize: '16px',
    paddingLeft: theme.spacing(5),
    paddingRight: theme.spacing(5),
    paddingTop: theme.spacing(3),
    paddingBottom: theme.spacing(3),
    border: 'none',
    cursor: 'pointer',
    borderRadius: '5px',
    textAlign: 'center'
  }

})

const BookDiscover: React.FC = () => {
  const imgSrc="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/296A1994-087A-4F86-BC94-D8E3D96946EE.png";
  const name="Discover";

  const classes = useStyles()
  return (
    <Grid className={classes.container}>
      <img src={imgSrc} alt="Snow" />
      <Button className={classes.btn} > {name}</Button>
    </Grid>
    
  )
}

export default BookDiscover