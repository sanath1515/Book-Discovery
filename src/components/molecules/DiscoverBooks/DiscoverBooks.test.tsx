import { render} from "@testing-library/react";
import BookDiscover from './BookDiscover'


describe("Topics renders correctly",()=>
{
    test("should match shapshot",()=>{
    const{asFragment}=render(<BookDiscover />);
    expect(asFragment()).toMatchSnapshot()

    })
})