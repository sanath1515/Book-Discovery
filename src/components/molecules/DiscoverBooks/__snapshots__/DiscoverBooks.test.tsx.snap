// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topics renders correctly should match shapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root makeStyles-container-1"
  >
    <img
      alt="Snow"
      src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/296A1994-087A-4F86-BC94-D8E3D96946EE.png"
    />
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-btn-2"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
         Discover
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
